packages/api/src/users/users.types.ts

```diff
@@ -1,6 +1,6 @@
 import { User } from '@coderscamp/shared/models/user';
 
-export type UserFromGithub = Pick<User, 'fullName' | 'githubId' | 'email' | 'image'>;
+export type UserFromGithub = Pick<User, 'fullName' | 'email' | 'image'>;
```

Comment:
shouldn't be 'UserFromGithub' renamed?  🤔

Reply:
No, it should be deleted 😄